packages/auth-providers-web/ambient.d.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-var */
+import type { HelmetServerState } from 'react-helmet-async'
+
+declare global {
+  var bazinga: boolean
+  var __REDWOOD__PRERENDERING: boolean
+  var __REDWOOD__HELMET_CONTEXT: { helmet?: HelmetServerState }
+  var __REDWOOD__APP_TITLE: string
+
+  // Provided by Vite.config in the user's project
+  var RWJS_ENV: {
+    RWJS_API_GRAPHQL_URL: string
+    /** URL or absolute path to the DbAuth serverless function */
+    RWJS_API_DBAUTH_URL: string
```

Comment:
This shouldn't be here. Created this PR to clean it up https://github.com/redwoodjs/redwood/pull/6968

Reply:
yeah, I'll take it out too! Do we need this ambient file in that case?

Reply:
I don't think so. But things also worked previously when we didn't have it, even though maybe we should have. Does Vite change something that makes the ambient files more important?